bazel-mode.el

```diff
@@ -0,0 +1,167 @@
+;;; bazel-mode.el --- Emacs major mode for editing Bazel BUILD and WORKSPACE files
+
+;; Copyright (C) 2018 Google LLC
+;; Licensed under the Apache License, Version 2.0 (the "License");
+;; you may not use this file except in compliance with the License.
+;; You may obtain a copy of the License at
+;;
+;;      http://www.apache.org/licenses/LICENSE-2.0
+;;
+;; Unless required by applicable law or agreed to in writing, software
+;; distributed under the License is distributed on an "AS IS" BASIS,
+;; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+;; See the License for the specific language governing permissions and
+;; limitations under the License.
+;;
+;;; Commentary:
+;;
```

Comment:
Please add a high-level description of the package here

Reply:
Done.

Reply:
The first line should be properly formatted now.